modules/installation-vsphere-machines.adoc

```diff
@@ -14,31 +14,56 @@ To install {product-title} on user-provisioned infrastructure on VMware vSphere,
 
 .Prerequisites
 
-* Obtain the Ignition config files for your cluster.
-* Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
+* You have obtained the Ignition config files for your cluster.
+* You have created a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
+* You have installed the VMWare PowerCLI (`vm`) command.
 
 .Procedure
 
-. Convert the control plane, compute, and bootstrap Ignition config files to Base64 encoding.
-
+. Locate the following Ignition config files that the installation program created, for example:
++
+* `<installation_directory>/bootstrap.ign`
+* `<installation_directory>/master.ign`
+* `<installation_directory>/worker.ign`
 +
-For example, if you use a Linux operating system, you can use the `base64` command to encode the files.
+. Use the PowerCLI (`vm`) utility to convert the Ignition config files to Base64 encoding and store them in the extra configuration properties of the virtual machine (VM) before booting by running the following commands:
++
+[source,terminal]
+----
+$ vm = New-VM -VM $Template -Name $name -ResourcePool $rp \
+  -Datastore $datastore -Location $Folder
```

Comment:
Per my understanding,  the purpose is to use PowerCLI to set bootstrap based64 encoded ignition config to extra config `guestinfo.ignition.config.data.encoding`, other parts including cloning vm from template, customized hardware etc still need to do through vSphere client, or you plan to change by using PowerCLI also for those parts?

Reply:
This is my understanding as well. PowerCLI is just used in this step to convert Ignition config files to Base64 encoding. I don't plan to change other parts of this doc to use PowerCLI. @jcpowermac can you confirm?